File: C2109L/ADF1/20-09-2022/javaPE01/src/main/java/com/aptech/models/Pizza.java
package com.aptech.models;

public class Pizza {
    private Double diameter;
    private Integer slices;

    public Pizza() {

    }

    public Pizza(Double diameter, Integer slices) {
        this.diameter = diameter;
        this.slices = slices;
    }

    @Override
    public String toString() {
        return "diameter : "+this.diameter+", "+
                "slices : "+this.slices+", ";
    }
}

File: C2108G2/AP-C#/02-02-2023/ConsoleApp/Models/Pizza.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp.Models
{
    public class Pizza
    {
        private float diameter;
        private int numberOfSlices;

        public float Diameter
        {
            get { return diameter; }
            set { diameter = value; }
        }

        public int NumberOfSlices
        {
            get { return numberOfSlices; }
            set { numberOfSlices = value; }
        }

        public override string ToString()
        {
            return "Diameter: " + diameter 
                + " inches" + "\n" +
                "Number of Slices: " + numberOfSlices;
        }
    }
}
